contracts/FlowToken.cdc

```diff
@@ -1,6 +1,9 @@
-import FungibleToken from 0xFUNGIBLETOKENADDRESS
+import FungibleToken from "FungibleToken"
+import MetadataViews from "MetadataViews"
+import FungibleTokenMetadataViews from "FungibleTokenMetadataViews"
+import ViewResolver from "ViewResolver"
```

Comment:
This applies elsewhere as well, but is there an issue with using quotes-only imports? I'm guessing it comes down to sdk support?

Reply:
I want to use quotes only imports, but I didn't want to do all of it in this PR, so I'll be doing the rest of it in a future PR